src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/MrdTestDataBuilder.kt

```diff
@@ -39,6 +42,15 @@ class MrdTestDataBuilder {
           ),
           responseToProbation = "They have not responded well",
           isThisAnEmergencyRecall = true,
+          victimsInContactScheme = VictimsInContactScheme(
+            selected = VictimsInContactSchemeValue.YES,
+            allOptions = listOf(
+              TextValueOption(value = "YES", text = "Yes"),
+              TextValueOption(value = "NO", text = "No"),
+              TextValueOption(value = "NOT_APPLICABLE", text = "Not applicable")
```

Comment:
Elsewhere in this PR, not applicable had the value N/A
https://github.com/ministryofjustice/make-recall-decision-api/pull/297/files#diff-ee153c8b8a6437d3c2e360446ab01ce1287ed05594418e6d799b9d8808efff37R13

Reply:
Yeh I can update, it's only test data though so doesn't really matter

Reply:
Fixed, with bit of a refactor